docs/guides/sap/odata-connector.md

```diff
@@ -0,0 +1,184 @@
+---
+id: odata-connector
+title: SAP OData Connector
+description: "The SAP OData Connector is a protocol and outbound Connector that runs as a Docker image on the SAP Business Technology Platform (BTP)."
+---
+
+The SAP OData Connector is a protocol and outbound [Connector](/components/connectors/introduction.md) that runs as a Docker image on the SAP Business Technology Platform (BTP).
+
+This Connector is designed to run in [hybrid mode](/guides/use-connectors-in-hybrid-mode.md), living in the customer's BTP sub-account in the Cloud Foundry environment.
+
+This Connector works with Camunda 8 SaaS, and utilizes BTP's [Destination](https://learning.sap.com/learning-journeys/administrating-sap-business-technology-platform/using-destinations) and [Connectivity](https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/what-is-sap-btp-connectivity) concepts to query a SAP system via both OData v2 and v4.
+
+:::note Important!
+This Connector is an alpha feature and available upon request. Visit [https://camunda.com/contact](https://camunda.com/contact) to contact us.
+:::
+
+## Quickstart
+
+For a standard overview of the steps involved in the SAP OData Connector, examine the diagram below:
+
+![OData steps](./img/odata-steps.png)
+
+## Prerequisites
+
+This Connector integrates into a standard BTP landscape and doesn't require any proprietary setup.
+
+However, a minimum set of SAP infrastructure requirements must be in place for the SAP OData Connector Docker image to run:
```

Comment:
Is this level of specificity (both "standard landscape/no proprietary setup" _and_ "minimum requirements") required? Is it possible to simply limit this to "To run the SAP OData Connector Docker image, the following SAP infrastructure setup is required:"